pkg/image/qemu.go

```diff
@@ -297,9 +297,16 @@ func (o *qemuOperations) CreateBlankImage(dest string, size resource.Quantity, p
 		return errors.Wrap(err, fmt.Sprintf("could not create raw image with size %s in %s", size.String(), dest))
 	}
 	// Change permissions to 0660
-	err = os.Chmod(dest, 0660)
+	file, err := os.Open(dest)
+	if err != nil {
+		return errors.Wrap(err, "could not get file descriptor for chmod call")
+	}
+	if err := file.Sync(); err != nil {
```

Comment:
so fsync first and then chmod? or the other way around

Reply:
definitely the other way, thx